src/app/components/Transactions/TransactionLink.tsx

```diff
@@ -8,22 +8,27 @@ import useMediaQuery from '@mui/material/useMediaQuery'
 import { TrimLinkLabel } from '../TrimLinkLabel'
```

Comment:
Also 
https://github.com/oasisprotocol/explorer/blob/da4bde6ad7aed9b0e4cc5164e1f3c7a902ed2d95/src/app/pages/SearchResultsPage/useRedirectIfSingleResult.ts#L22-L25

https://github.com/oasisprotocol/explorer/blob/da4bde6ad7aed9b0e4cc5164e1f3c7a902ed2d95/src/app/pages/SearchResultsPage/index.tsx#L132

Reply:
OK, no we are handling search, too.